include/matrix3x4SIMD.h

```diff
@@ -389,25 +408,28 @@ struct matrix3x4SIMD
 		const __m128 mask1110 = _mm_castsi128_ps(_mm_setr_epi32(0xffffffff, 0xffffffff, 0xffffffff, 0));
 		vectorSIMDf col3;
 
-		vectorSIMDf tmp = (-vectorSIMDf(r0) & mask1110) * center;
-		__m128 ttmp = tmp.getAsRegister();
-		ttmp = _mm_hadd_ps(ttmp, ttmp);
-		ttmp = _mm_hadd_ps(ttmp, ttmp);
-		col3.x = vectorSIMDf(ttmp).x;
-
-		tmp = (-vectorSIMDf(r1) & mask1110) * center;
-		ttmp = tmp.getAsRegister();
-		ttmp = _mm_hadd_ps(ttmp, ttmp);
-		ttmp = _mm_hadd_ps(ttmp, ttmp);
-		col3.y = vectorSIMDf(ttmp).x;
-
-		tmp = (-vectorSIMDf(r2) & mask1110) * center;
-		ttmp = tmp.getAsRegister();
-		ttmp = _mm_hadd_ps(ttmp, ttmp);
-		ttmp = _mm_hadd_ps(ttmp, ttmp);
-		col3.z = vectorSIMDf(ttmp).x;
-
-		col3 += (center - translation) & mask1110;
+		//vectorSIMDf tmp = (-vectorSIMDf(r0) & mask1110) * _center;
+		//__m128 ttmp = tmp.getAsRegister();
+		//ttmp = _mm_hadd_ps(ttmp, ttmp);
+		//ttmp = _mm_hadd_ps(ttmp, ttmp);
+		//col3.x = vectorSIMDf(ttmp).x;
+
+		//tmp = (-vectorSIMDf(r1) & mask1110) * _center;
+		//ttmp = tmp.getAsRegister();
+		//ttmp = _mm_hadd_ps(ttmp, ttmp);
+		//ttmp = _mm_hadd_ps(ttmp, ttmp);
+		//col3.y = vectorSIMDf(ttmp).x;
+
+		//tmp = (-vectorSIMDf(r2) & mask1110) * _center;
+		//ttmp = tmp.getAsRegister();
+		//ttmp = _mm_hadd_ps(ttmp, ttmp);
+		//ttmp = _mm_hadd_ps(ttmp, ttmp);
+		//col3.z = vectorSIMDf(ttmp).x;
+		
+		//result slightly (but significantly) differ from above.. why?
+		col3 = (-vectorSIMDf(c0) - c1 - c2) * _center;
+
+		col3 += (_center - _translation) & mask1110;
```

Comment:
there's no need for the mask as the final component doesn't get saved as there is no row[3]